src/databricks/labs/remorph/transpiler/execute.py

```diff
@@ -158,7 +169,7 @@ async def _process_input_dir(config: TranspileConfig, validator: Validator | Non
     for source_dir, _, files in dir_walk(input_path):
         relative_path = cast(Path, source_dir).relative_to(input_path)
         transpiled_dir = output_folder / relative_path
-        logger.debug(f"Transpiling sql files from folder: {source_dir!s} into {transpiled_dir!s}")
+        logger.debug(f"Transpiling SQL files from folder: {source_dir} -> {transpiled_dir}")
```

Comment:
```suggestion
        logger.debug(f"Transpiling files from folder: {source_dir} -> {transpiled_dir}")
```

we also do infa and datastage files.